### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Databases for assessment_Keyboard: User is unable to access Deselect/select All checkbox using keyboard in databases grid in "Databases for assessment" step 

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19831]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG2.1.1; #Keyboard; 

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Repro Steps:

1. Install Azure data studio app and install Azure SQL migration extension
2. Connect to SQL server, tab till respective server, right click and select manage
3. Tab till Azure SQL Migration in sitemap and press enter key
4. Tab till migrate to azure sql and press enter
5. Navigate to databases grid in "Databases for assessment" in step 1.
6. Verify whether user is able to access Deselect/select All databases checkbox or not

### Actual Result:
User is unable to access Deselect/select All checkbox using keyboard in databases grid in "Databases for assessment" step 

### Expected Result:
User should be able to access Deselect/select All checkbox using keyboard in databases grid in "Databases for assessment" step 

### Attachment:
![Keyboard](https://user-images.githubusercontent.com/83570638/176096584-f4074472-17dc-47cb-9c1d-ad47ba68acdf.gif)

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed, hence closing the bug. deselect/select All checkbox is accessible using keyboard in databases grid in "Databases for assessment" step. 
![19831_fixed](https://user-images.githubusercontent.com/83570638/192482110-87b4dcb6-e40a-4eac-8a9a-3c4e0a4c2a5e.gif)

**Comment by VijayaBharathyA** [comment]
#Closed;
